### ISSUE TITLE
Clients Apps improvements

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#1160]
Following from discussion in https://github.com/freedombox/Plinth/pull/1112:
- [ ] Change desktop/mobile buttons color to indicate that they are links.
- [ ] Consider re-arrangement of clients for usability and space efficiency.

**Comment by JoKeyser** [comment]
For easy overview, here an example of the current state of Syncthing:
![syncthingclientapps](https://user-images.githubusercontent.com/8722846/34469475-a3832848-ef17-11e7-8a6e-70cfe87cc2d6.png)

**Comment by JoKeyser** [comment]
About the coloring sub-issue, I'm not sure whether *"Change desktop/mobile buttons color to indicate that they are links"* is what really follows from the previous discussions...
My reading of #1112 (and #1177) is that @SunilMohanAdapa argues that the Web `Launch` button should be emphasized because it has a different role than the other client buttons (which are just links). It's meant to launch the Freedomboxes' own, local client application to the service in question.
On the other hand, @mray argues that the green color is not enough to clarify this special role of the Launch button, e.g. it might be confused with Web-clients hosted elsewhere.
@mray did you mean that all buttons should be blue?

**Comment by jvalleroy** [comment]
Just to add one minor suggestion. When expanding / collapsing "Client Apps", the icons get rearranged after a very short time (<1 second). Is there a way to make this quick enough to not be visible? Otherwise it is just some distracting movement on the page.

**Comment by SunilMohanAdapa** [comment]
@jvalleroy, I have a feeling we are doing something wrong the way we are using the collapse function that is causing the incorrect resize and animation. We should investigate and fix it. I added this to TODO.

**Comment by SunilMohanAdapa** [comment]
@mray , @JoKeyser , I agree that we should clearly distinguish links opening to outside and links open within FreedomBox interface. For this could adopt what Wikipedia does for this: all links opening external websites will be marked with an special icon next to them. We have this icon on the 'Launch' button now, we can remove this and add it instead to every link that is going outside.

**Comment by mray** [comment]
@JoKeyser @SunilMohanAdapa 

I think there should only be one link key-color, external links should be distinguished by the mentioned icon. My mockups were just based on the provided prior mockups.
